tests/resume.test.ts

```diff
@@ -21,6 +21,35 @@ afterAll(async () => {
   await DatabaseConnection.close();
 });
 
+describe('resume fetching', () => {
+  test('only admin can fetch and only the visible resumes', async () => {
+    const conn = await DatabaseConnection.get();
+    const admin = UserFactory.fake({ accessType: UserAccessType.ADMIN });
+    const member = UserFactory.fake();
+
+    await new PortalState()
+      .createUsers(admin, member)
+      .write();
+
+    const resume = FileFactory.pdf(Config.file.MAX_RESUME_FILE_SIZE / 2);
+    const fileLocation = 'fake location';
+
+    const storageService = Mocks.storage(fileLocation);
+    const resumeController = ControllerFactory.resume(conn, instance(storageService));
+    const uploadedResume = (await resumeController.updateResume(resume, member)).resume;
+    delete uploadedResume.user;
```

Comment:
This resume upload operation can be moved into PortalState. The reason we have PortalState is to make it easy to persist fake data into the database.

Here, we are creating mocks, controllers, etc. to upload a resume but all we care about is that a resume is in the database - we've already validated that the 'upload resume' controller works as intended. So instead, let's modify PortalState to have a `createResumes` function that takes in a user and a list of resumes, and uploads resumes for that user. Note that we'd also need a `ResumeFactory` to create fake resumes.

The corresponding code should look something like this:
```ts
    const conn = await DatabaseConnection.get();
    const admin = UserFactory.fake({ accessType: UserAccessType.ADMIN });
    const member = UserFactory.fake();
    const resume = ResumeFactory.fake({ userId: member.id });

    await new PortalState()
      .createUsers(admin, member)
      .createResumes(member, resume)
      .write();
    
    // throws permissions error for member
    expect(resumeController.getAllVisibleResumes(member))
      .rejects.toThrow(ForbiddenError);

    // get response resumes
    const response = await resumeController.getAllVisibleResumes(admin);
    expect(response.error).toBeNull();
    expect(response.resumes).toContainEqual(uploadedResume);
```

Let me know if there's any questions with what I mentioned!